README.rst

```diff
@@ -1,70 +1,54 @@
 Description
-===========
+-----------
 
 Generalized all-in-one parameters module.
 
-This package provides a single interface to environment variables', 
-configuration files', and command line arguments' provided values.  The 
-dictionary-like interface makes interacting with these, most of the time 
-disparate, resources much simpler.  It also allows parameters' values to be set
-in any of the three sources and selects an appropriate value when a parameter's
-value is specified in multiple sources.  This way the most expected value,
-according to the normal prcedence—command line arguments then configuration
-files then environment variables, is always returned.
+This package provides a single interface to environment variables,
+configuration files, and command line arguments.  The dictionary-like interface
+makes interacting with these disparate resources much simpler.  It also allows
+parameters' values to be set in any of the three sources and selects an
+appropriate value when a parameter's value is specified in multiple sources.
+This way the most expected value, according to the normal prcedence—command
```

Comment:
precedence 
Sentence reads awkwardly. 
Parameter value precedence is, in order: command line, environment variables, config files. 

Your order was different than mine. Use whichever is correct for your module.